afs-cassandra/src/test/java/com/powsybl/afs/cassandra/CassandraAppStorageTest.java

```diff
@@ -6,14 +6,15 @@
  */
```

Comment:
On that topic, I will propose powsybl team to just drop official support of cassandra storage on windows.

I have disabled the `CassandraAppStorageTest` on windows so that the CI passes.